### PR TITLE
Fix EK certificate generation

### PR DESCRIPTION
**Pull request by arturkow2000**

### CODE REVIEW

File: tools/tpm_manufacture.sh [hunk 1]
```diff
@@ -9,6 +9,7 @@ CA_SRL=$KNC/ca.srl
 EK_PUB=$KNC/ek_pub.pem
 EK_CSR=$KNC/ek.csr
 EK_CERT=$KNC/ek_cert.der
+EK_CTX=$KNC/ek_ctx.der
```
Comment (krystian-hebel) [inline tools/tpm_manufacture.sh]: This file is not removed. On the other hand, `tpm2 createek (...) -c /dev/null` also works and except for one used transient handle (cleared later anyway) doesn't have any bad side effects.

**Review by krystian-hebel — CHANGES_REQUESTED** [review]

File: tools/tpm_manufacture.sh (hunk 1 above)
Comment (arturkow2000) [reply tools/tpm_manufacture.sh]: Fixed.

File: tools/tpm_manufacture.sh [hunk 2]
```diff
@@ -75,7 +76,9 @@ openssl req -newkey rsa:2048 -nodes -keyout $CA_PRIV -x509 -days 365 \
 # Create and sign EK certificate
 openssl req -new -key $CA_PRIV -out $EK_CSR -config $EK_CONFIG
 openssl x509 -req -in $EK_CSR -CA $CA_CERT -CAkey $CA_PRIV -CAserial $CA_SRL \
-        -CAcreateserial -out $EK_CERT -outform der
+        -force_pubkey $EK_PUB \
```
Comment (krystian-hebel) [inline tools/tpm_manufacture.sh]: Can you move this around so that all CA parameters are grouped together?

Comment (arturkow2000) [reply tools/tpm_manufacture.sh]: Fixed

**Review by krystian-hebel — APPROVED** [review]

**Pull request merged**